apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -260,9 +260,11 @@ const NewsItem = new GQLListSchema('NewsItem', {
             }
 
             if (operation === 'update') {
-                if (sentAt) {
+                // You can update only validBefore field if news item was already sent
+                if (sentAt && (Object.keys(resolvedData) > 1 || isEmpty(get(resolvedData, 'validBefore') ))) {
```

Comment:
Should we use `resolvedData` or `originalInput` here?

Reply:
what about dv and sender?

Reply:
From the front-line utilities, the dv and sender are entered automatically, it seems.

Reply:
> dv and sender are entered automatically, it seems.

That's right, so they need to be filtered and take `originalInput` instead of `resolvedData`.
`resolvedData` can contain a lot of unnecessary stuff